go/vt/wrangler/vexec.go

```diff
@@ -331,8 +361,12 @@ func (wr *Wrangler) getStreams(ctx context.Context, workflow, keyspace string) (
 	if err != nil {
 		return nil, err
 	}
+
+	// We set a topo timeout since we contact topo for the shard record.
+	ctx, cancel := context.WithTimeout(ctx, 10*time.Second)
```

Comment:
I'd recommend using an existing flag. `*topo.RemoteOperationTimeout` looks appropriate.

Reply:
Good call, I'll swap it out.

Reply:
Swapped.